src/Viewer/components/modals/CalendarModal/DatetimePicker.jsx

```diff
@@ -18,7 +18,7 @@ import {
 const DateTimePicker = ({onChange, value}) => {
     const commonProps = {
         autoFocus: false,
-        timezone: "UTC",
+        timezone: "default",
```

Comment:
MUI X: Can not find the date and time pickers localization context.
It looks like you forgot to wrap your component in LocalizationProvider.
This can also happen if you are bundling multiple versions of the `@mui/x-date-pickers` package